notebooks/06_Serving/06_02_Kfserving.ipynb

```diff
@@ -0,0 +1,493 @@
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run a prediction\n",
+    "\n",
+    "Use `istio-ingressgateway` as your `INGRESS_GATEWAY` if you are deploying KFServing as part of Kubeflow install, and not independently.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MODEL_NAME=flowers-sample\n",
+    "INPUT_PATH=@./input.json\n",
+    "INGRESS_GATEWAY=istio-ingressgateway\n",
+    "CLUSTER_IP=$(kubectl -n istio-system get service $INGRESS_GATEWAY -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')\n",
+    "SERVICE_HOSTNAME=$(kubectl get inferenceservice ${MODEL_NAME} -n kubeflow -o jsonpath='{.status.url}' | cut -d \"/\" -f 3)\n",
+    "\n",
+    "curl -v -H \"Host: ${SERVICE_HOSTNAME}\" http://$CLUSTER_IP/v1/models/$MODEL_NAME:predict -d $INPUT_PATH"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Expected Output\n",
+    "```\n",
+    "*   Trying 34.83.190.188...\n",
+    "* TCP_NODELAY set\n",
+    "* Connected to 34.83.190.188 (34.83.190.188) port 80 (#0)\n",
+    "> POST /v1/models/flowers-sample:predict HTTP/1.1\n",
+    "> Host: flowers-sample.default.svc.cluster.local\n",
+    "> User-Agent: curl/7.60.0\n",
+    "> Accept: */*\n",
+    "> Content-Length: 16201\n",
+    "> Content-Type: application/x-www-form-urlencoded\n",
+    "> Expect: 100-continue\n",
+    "> \n",
+    "< HTTP/1.1 100 Continue\n",
+    "* We are completely uploaded and fine\n",
+    "< HTTP/1.1 200 OK\n",
+    "< content-length: 204\n",
+    "< content-type: application/json\n",
+    "< date: Fri, 10 May 2019 23:22:04 GMT\n",
+    "< server: envoy\n",
+    "< x-envoy-upstream-service-time: 19162\n",
+    "< \n",
+    "{\n",
+    "    \"predictions\": [\n",
+    "        {\n",
+    "            \"scores\": [0.999115, 9.20988e-05, 0.000136786, 0.000337257, 0.000300533, 1.84814e-05],\n",
+    "            \"prediction\": 0,\n",
+    "            \"key\": \"   1\"\n",
+    "        }\n",
+    "    ]\n",
+    "* Connection #0 to host 34.83.190.188 left intact\n",
+    "}%\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you stop making requests to the application, you should eventually see that your application scales itself back down to zero. Watch the pod until you see that it is `Terminating`. This should take approximately 90 seconds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl get pods --watch -n kubeflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: To exit the watch, use `ctrl + c`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Canary Rollout\n",
+    "\n",
+    "To test a canary rollout, you can use the tensorflow-canary.yaml \n"
```

Comment:
I think you need to have a similar prediction request section for this. Otherwise, how does user verify that?

Reply:
Sure, added in the new revision.